website/blog/2021-01-05-macromix.md

```diff
@@ -0,0 +1,184 @@
+---
+author: Ólafur Páll Geirsson
+title: Publish Scala 2 and Scala 3 macros together
+authorURL: https://twitter.com/olafurpg
+authorImageURL: https://github.com/olafurpg.png
+---
+
+The next release of MUnit makes use of a new compiler feature that allows you to
```

Comment:
cc/ @bishabosha @sjrd could you please take a look at this blog post? Please correct me if I got the details wrong

Reply:
Live preview at https://olafurpg.github.io/munit/blog/2021/01/05/macromix.html so you can see the rendered SVG files.